### PR TITLE
Development

### PR DESCRIPTION
**Pull request by micaylamendoza**
Screen Shot 2020-06-06 at 8 12 32 PM
Screen Shot 2020-06-06 at 8 12 58 PM
Screen Shot 2020-06-06 at 8 13 27 PM
I edited my homepage to have a welcome message and image. I created a navigation bar on the left side to access multiple pages. I have only created the html file for the about me page, but I haven't styled it yet. So no other pages, besides "Home" and "About Me," can be accessed from the navigation bar yet.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -6,13 +6,23 @@
...
+    <span style="font-size:30px;cursor:pointer" onclick="openNav()">&#9776;</span>  
+    
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Hey, hello!</h1><p>
+      <img src="images/Untitled_Artwork.png" width="700" height="525"/><p>
```
Comment (classm-sps) [inline portfolio/src/main/webapp/index.html]: A couple of things here:

- consider giving the file a more descriptive name
- I don't see the image being added to the repo

**Review by classm-sps — CHANGES_REQUESTED** [review]
Great PR description! It's a really good habit to have!

Have you deployed your server? (https://github.com/micaylamendoza/my-portfolio/blob/master/walkthroughs/week-1-web-development/portfolio-walkthrough.md#live-server)

I would expect additional file(s) to be modified if that were the case. Definitely worth investing some time up front to do that.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (micaylamendoza) [reply portfolio/src/main/webapp/index.html]: You don't see the .png being added?

**Review by classm-sps — APPROVED** [review]
LGTM

**Pull request merged**